pkg/workspace/template_loader.go

```diff
@@ -187,6 +187,8 @@ func (l *TemplateLoader) EvalYAML(libraryCtx LibraryExecutionContext, file *file
 	if err != nil {
 		return nil, nil, err
 	}
+
+	resultVal.(*yamlmeta.DocumentSet).SetRawBytes(docSet)
```

Comment:
this feels very weird. it's also not clear from the code why this is necessary.

Reply:
Removed by giving ownership of the original line content to Position.